pyintegration/tests/test_calendars.py

```diff
@@ -0,0 +1,42 @@
+#
+#   Copyright (c) 2016-2022 Deephaven Data Labs and Patent Pending
+#
+
+import unittest
+
+from deephaven2 import DHError
+from deephaven2.calendars import calendar_names, default_calendar_name, Calendar
+from tests.testbase import BaseTestCase
+
+
+class CalendarsTestCase(BaseTestCase):
+    def test_calendars(self):
+        with self.subTest(msg="calendarNames() test"):
+            names = calendar_names()
+            self.assertTrue(isinstance(names, list) and len(names) > 0)
+
+        with self.subTest(msg="getDefaultName() test"):
+            name = default_calendar_name()
+            self.assertIsNotNone(name)
+
+        with self.subTest(msg="calendar() construction"):
+            self.assertIsNotNone(Calendar())
+
+        with self.subTest(msg="calendar(invalid name) construction"):
+            self.assertRaises(DHError, Calendar, "garbage_name")
+
+        with self.subTest(msg="calendar(valid name) construction"):
+            self.assertIsNotNone(Calendar(name))
```

Comment:
Are you intending to call this with `default_calendar_name()`?  Just asking, because it took me a long time to find where the name variable was defined.